src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -28,11 +30,14 @@ const VirtualAccOpening = (() => {
 
     const init = () => {
         $(form).setVisibility(1);
+        const mobileDevices = ['iPhone', 'iPad', 'Android'];
```

Comment:
for variable names we use underscore, you can check https://github.com/binary-com/binary-static/tree/master/src/javascript#naming-conventions